src/types.rs

```diff
@@ -43,11 +43,11 @@ pub(crate) fn rewrite_path(
 ) -> Option<String> {
     let skip_count = qself.as_ref().map_or(0, |x| x.position);
 
-    let mut result = if path.is_global() && qself.is_none() && path_context != PathContext::Import {
-        "::".to_owned()
-    } else {
-        String::new()
-    };
+    let mut result = String::with_capacity(32);
```

Comment:
nit: might be worth adding a comment explaining why 32 was chosen as the initial capacity. I could see someone reading this in the future and wondering about that.

Reply:
sounds good! people can still `git blame`/`git log -L` their way to this PR which contains more info, but a quick comment makes sense too.